assignment_2_Script.js

```diff
@@ -0,0 +1,81 @@
+var dateCreation = (function iife() {
+  var currentModuleDate = null;
+  function instantiateDate() {
+    return new Date();
+  }
+  function setDate(date) {
+    if (date===undefined) {
+      currentModuleDate = instantiateDate();
+    }
+    else if (typeof date === 'number') {
+      currentModuleDate = new Date(date);
+    }
+    else if (date instanceof Date) {
+      currentModuleDate = new Date(date);
+    }
+  }
+  function getDate(requestedFormat) {
+    if (currentModuleDate === null) {
+      currentModuleDate = instantiateDate();
+    }
+    if (requestedFormat.format === 'formatted') {
+      return currentModuleDate.getMonth() + ' ' + currentModuleDate.getDate() +
```

Comment:
The output should look something like "May 4, 2016" but calling `getMonth()` will return 4 here instead "May", so you'll need a different method call here. You've implemented another function that will return the string you want...